src/Microsoft.HttpRepl/ApiConnection.cs

```diff
@@ -48,11 +31,12 @@ internal class ApiConnection
         public bool HasSwaggerDocument => SwaggerDocument is object;
         public bool AllowBaseOverrideBySwagger { get; set; }
 
-        public ApiConnection(IPreferences preferences, IWritable logger, bool logVerboseMessages)
+        public ApiConnection(IPreferences preferences, IWritable logger, bool logVerboseMessages, IOpenApiSearchPathsProvider? openApiSearchPaths = null)
         {
-            _preferences = preferences ?? throw new ArgumentNullException(nameof(preferences));
+            _ = preferences ?? throw new ArgumentNullException(nameof(preferences));
```

Comment:
I'm assuming the "looks a little funky" was in reference to using the discard. According to Jimmy, [it saves a few ops!](https://github.com/dotnet/HttpRepl/pull/417#discussion_r512501923)

But yeah, since I also changed it to not store the value anymore and just pass it, moving it makes sense. Done.